src/hooks/usePoolsData.ts

```diff
@@ -37,7 +37,15 @@ const usePoolsData = () => {
       });
 
       const poolsData: PoolsData = await poolsDataResponse.json();
-      return poolsData.pools.filter(poolData => poolData.details !== null);
+      return poolsData.pools
+        .filter(poolData => {
```

Comment:
This filtering was already covered in previous pull request:
https://github.com/mira-amm/mira-amm-web/blob/778b306112583b0e3a3eeb011638b546890114c5/src/hooks/usePoolsData.ts#L41

I guess that's the place where merge conflicts appear now